teammapper-backend/src/map/services/maps.service.ts

```diff
@@ -233,19 +239,10 @@ export class MapsService {
         'lastmodifiednode.nodeMapid = map.id'
       )
       .where(
-        // delete all maps that have nodes that were last updated after afterDays
-        "(lastmodifiednode.lastUpdatedAt + (INTERVAL '1 day' * :afterDays)) < :today",
+        "(GREATEST(map.lastAccessed, map.lastModified, lastmodifiednode.lastUpdatedAt) + (INTERVAL '1 day' * :afterDays)) < :today",
         { afterDays, today }
       )
-      .orWhere(
-        new Brackets((qb) => {
-          // also delete empty maps, use th emaps lastmodified date for this:
-          qb.where('lastmodifiednode.lastUpdatedAt IS NULL').andWhere(
-            "(map.lastModified + (INTERVAL '1 day' * :afterDays)) < :today",
-            { afterDays, today }
-          )
-        })
-      )
+    // One case left: If lastModified is set but lastAccessed isn't and lastUpdatedAt is null, map should be deleted
```

Comment:
so this is a todo?

Reply:
I accidentally left that in there, that's covered now.